source/module_hsolver/diago_david.cpp

```diff
@@ -547,6 +558,16 @@ void DiagoDavid::diag_zhegvx(const int &n,
     delete[] rwork;
     delete[] iwork;
     delete[] ifail;
+}
+
+#ifdef __MPI
+    for (int i=0;i<n;i++)
+    {
+        Parallel_Common::bcast_complex_double(&vc(i, 0), m);
```

Comment:
What's the point not broadcasting for `vc` in one pass?

Reply:
only partial of the matrix are needed to do bcast, the full matrix is much larger than m*n